service/service.py

```diff
@@ -155,6 +155,22 @@ def index():
 )
 
 
+# query string arguments NOTE: Whoever is doing the query ticket should probably uncomment these and
+# try to make it work with the list function in the promotion collection
+# --------------------------------------------------------------------------------------------------
+# promotion_args = reqparse.RequestParser()
+# promotion_args.add_argument('title', type=str, required=False, help='List Promotions by title')
+# promotion_args.add_argument('promo_code', type=str, required=False, help='List Promotions by promo code')
+# promotion_args.add_argument('promo_type', type=str, required=False, help='List Promotions by promo type')
+# promotion_args.add_argument('amount', type=int, required=False, help='List Promotions by the amount discounted')
+# promotion_args.add_argument('start_date', type=str, required=False, help='List Promotions by start date')
+# promotion_args.add_argument('end_date', type=str, required=False, help='List Promotions by end date')
+# promotion_args.add_argument('duration', type=int, required=False, help='List Promotions by duration')
+# promotion_args.add_argument('active', type=str, required=False, help='List Promotions by active status')
+# promotion_args.add_argument('is_site_wide', type=str, required=False, help='List Promotions by site wide status')
+# promotion_args.add_argument('product', type=int, required=False, help='List Promotions by a product')
```

Comment:
This is all the stuff that adds documentation for how to run queries, it, combined with the commented out decorator should be part of the solution to adding the query stuff to swagger